pkl/project_test.go

```diff
@@ -108,7 +119,7 @@ func TestLoadProject(t *testing.T) {
 
 		t.Run("evaluatorSettings", func(t *testing.T) {
 			fals := false
-			expectedSettings := &pkl.ProjectEvaluatorSettings{
+			expectedSettings := &pkl.EvaluatorSettings{
```

Comment:
Two issues here (per failing tests):

1. The evaluator settings should have `Http`
2. Rewrite test to pass on Pkl 0.25 (probably need different tests for the different versions)